app/js/components/manageApps/ManageApps.jsx

```diff
@@ -699,7 +699,8 @@ export default class ManageApps extends React.Component {
       });
     }).catch(
       (error) => {
-        toastr.error(error);
+        error.message === "Network Error" ? toastr.error("There is no internet connection") : toastr.error(error);
```

Comment:
Isn't `error` in `toastr.error(error);` an object? There is a better way to deal with this kind error above. Have a look [here](https://github.com/axios/axios#handling-errors). In particular, inspect the error and decide what to show.